src/main/resources/application.properties

```diff
@@ -1 +1,11 @@
 spring.application.name=demo
+
+# Database Configuration
```

Comment:
Use environment variables for these database settings. Prefix them with PROJECT-1-